### ISSUE TITLE
[Row Layout] Responsive column gutter width.

### ISSUE THREAD

**Issue by KonradDeskiewicz** [kadencewp/kadence-blocks#128]
It would be nice to have an option for the different gutter sizes on tablet/mobile if no column stacking.
At this moment we need to use lower gutter width and add artificial padding on aligned content on a larger screen. Workaround but not an elegant one.

**Comment by OLEXNY** [comment]
Agreed, individual gutter settings for mobile viewports are missing and is an essential feature.
